src/Logger/AdapterFactory.php

```diff
@@ -47,7 +37,7 @@ public function __construct(array $services = [])
      * @param array  $options
      *
      * @return AdapterInterface
-     * @throws ExceptionAlias
+     * @throws SupportException
```

Comment:
Will a namespace alias work with the doc's generation?

Reply:
I believe it does

Reply:
Yes it does